demos/handwritten_text_recognition_demo/python/handwritten_text_recognition_demo.py

```diff
@@ -48,6 +48,7 @@ def build_argparser():
                       help="Path to the decoding char list file. Default is for Japanese")
     args.add_argument("-dc", "--designated_characters", type=str, default=None, help="Optional. Path to the designated character file")
     args.add_argument("-tk", "--top_k", type=int, default=20, help="Optional. Top k steps in looking up the decoded character, until a designated one is found")
+    args.add_argument("-ob", "--output_blob", type=str, default="output", help="Optional. Name of the output layer of the model. Default is \'output\'")
```

Comment:
I think it will be better if default behavior to get output from model instead hardcoding name (it preserves previous demo behavior and allow to run custom models in the demo, where single output, but name is different)

Reply:
I've updated the script to reuse previous code unless this value is set to not `None`.